protos/tracking_server/tracking_server.proto

```diff
@@ -0,0 +1,106 @@
+service TrackingServerService {
+    // Set/update the current point tracking status.
+    rpc SetTrackingPointStatus(SetTrackingPointStatusRequest) returns(SetTrackingPointStatusResponse) { option (mavsdk.options.async_type) = SYNC; }
+    // Set/update the current rectangle tracking status.
+    rpc SetTrackingRectangleStatus(SetTrackingRectangleStatusRequest) returns(SetTrackingRectangleStatusResponse) { option (mavsdk.options.async_type) = SYNC; }
+    // Set the current tracking status to off.
+    rpc SetTrackingOffStatus(SetTrackingOffStatusRequest) returns(SetTrackingOffStatusResponse) { option (mavsdk.options.async_type) = SYNC; }
+    // Subscribe to incoming tracking point command.
+    rpc SubscribeTrackingPointCommand(SubscribeTrackingPointCommandRequest) returns(stream TrackingPointCommandResponse) { option (mavsdk.options.async_type) = ASYNC; }
+    // Subscribe to incoming tracking rectangle command.
+    rpc SubscribeTrackingRectangleCommand(SubscribeTrackingRectangleCommandRequest) returns(stream TrackingRectangleCommandResponse) { option (mavsdk.options.async_type) = ASYNC; }
+    // Subscribe to incoming tracking off command.
+    rpc SubscribeTrackingOffCommand(SubscribeTrackingOffCommandRequest) returns(stream TrackingOffCommandResponse) { option (mavsdk.options.async_type) = ASYNC; }
+    // Respond to an incoming tracking point command.
+    rpc RespondTrackingPointCommand(RespondTrackingPointCommandRequest) returns(RespondTrackingPointCommandResponse) { option (mavsdk.options.async_type) = SYNC; }
+    // Respond to an incoming tracking rectangle command.
+    rpc RespondTrackingRectangleCommand(RespondTrackingRectangleCommandRequest) returns(RespondTrackingRectangleCommandResponse) { option (mavsdk.options.async_type) = SYNC; }
+    // Respond to an incoming tracking off command.
+    rpc RespondTrackingOffCommand(RespondTrackingOffCommandRequest) returns(RespondTrackingOffCommandResponse) { option (mavsdk.options.async_type) = SYNC; }
+}
+
+message SetTrackingPointStatusRequest {
+    TrackPoint tracked_point = 1; // The tracked point
+}
+message SetTrackingPointStatusResponse {}
+
+message SetTrackingRectangleStatusRequest {
+    TrackRectangle tracked_rectangle = 1; // The tracked rectangle
+}
+message SetTrackingRectangleStatusResponse {}
+
+message SetTrackingOffStatusRequest {}
+message SetTrackingOffStatusResponse {}
+
+message SubscribeTrackingPointCommandRequest {}
+message TrackingPointCommandResponse {
+    TrackPoint track_point = 1; // The point to track if a point is to be tracked
+}
+
+message SubscribeTrackingRectangleCommandRequest {}
+message TrackingRectangleCommandResponse {
+    TrackRectangle track_rectangle = 1; // The point to track if a point is to be tracked
+}
+
+message SubscribeTrackingOffCommandRequest {}
+message TrackingOffCommandResponse {}
+
+message RespondTrackingPointCommandRequest {
+    TrackingServerResult result = 1; // The result to answer to the incoming command
```

Comment:
I think it could be fine, but it could also be that the generator expects it to be called `tracking_server_result` instead of `result`. But you'll see it early if it complains :see_no_evil:

Reply:
Oh, right, I'll change it.

Reply:
I think that would be for the response though, not the result.